lisp/ess-custom.el

```diff
@@ -2321,41 +2325,88 @@ and `ess-tracebug'.")
 
 ;;*;; Inferior ESS commands
 
-(defvar ess-load-command "source(\"%s\")\n"
+(defvar ess-load-command "source('%s')\n"
   "Dialect specific format-string for building the ess command to load a file.
 
 This format string should use %s to substitute a file name and should
 result in an ESS expression that will command the inferior ESS to load
 that file.")
-(define-obsolete-variable-alias 'inferior-ess-load-command 'ess-load-command "ESS v13.09")
-
-(defvar ess-load-visibly-command nil
```

Comment:
There was not much point in multiplying the number of variables for each different parameter. There are now two possibilities: using a simple `ess-load/eval-command`, or writing a formatting function and binding it to `ess-format-load/eval-command-function` in the customise-alist. That function should be in charge of parameterising the command. The function has priority over the variable when both are defined.